sdk/storagecache/Microsoft.Azure.Management.StorageCache/tests/Fixtures/StorageAccountsFixture.cs

```diff
@@ -36,13 +36,17 @@ public class StorageAccountsFixture : IDisposable
         /// </summary>
         private readonly Dictionary<string, BlobContainer> blobContainersCache = new Dictionary<string, BlobContainer>();
 
+        public List<string> notes;
+
         /// <summary>
         /// Initializes a new instance of the <see cref="StorageAccountsFixture"/> class.
         /// </summary>
         /// <param name="fixture">StorageCacheTestFixture.</param>
         public StorageAccountsFixture(StorageCacheTestFixture fixture)
         {
+            notes = new List<string>();
             this.fixture = fixture;
+            notes.Add("SAF Ctor");
```

Comment:
Its seems this is for debug purpose and can be removed as we use testOutputHelper ?

Reply:
Update message in that. 
Yeah it was for logging.